README: update `bump diff` command since we allow public diffs now (#275)
bump-sh/cli: README.md

_If you want to receive automatic `bump diff` results on your Github Pull Requests you might be interested by [our Github Action](https://github.com/marketplace/actions/api-documentation-on-bump#api-diff-on-pull-requests) diff command._

#### Public API diffs

> From any two API definition files or URLs you can retrieve a comprehensive changelog of what has changed between the two given definitions.

Author: C'est nul !

> From any two API definition files or URLs you can retrieve a comprehensive changelog of what has changed between the two given definitions.

Reviewer: What about:
`From any two definition files or URLs, you can retrieve a comprehensive changelog of what has changed between them.` ?

Author: ```suggestion
From any two definition files or URLs, you can retrieve a comprehensive changelog of what has changed between them.
```
Indeed, thanks!

```sh-session
$ bump diff path/to/your/file.yml path/to/your/second_file.yml
* Comparing the two given definition files... done
Modified: GET /consommations
  Response modified: 200
    [Breaking] Body attribute modified: energie
```

Or from two URLs:

```sh-session
$ bump diff https://demo.bump.sh/doc/trips-books/changes/bfec0a43-b870-44da-9e07-60c8955e15d5.json https://demo.bump.sh/doc/trips-books.json
* Comparing the two given definition files... done
Modified: POST /books
  Response modified: 200
    [Breaking] Body attribute removed: cent
```

> _Note: You can also test this feature in our dedicated web application at <https://api-diff.io/>._

Author: C'est pas bien ou c'est bien
